builders/tools/explorers/verify-contract.md

```diff
@@ -0,0 +1,131 @@
+
+    ```
+    // SPDX-License-Identifier: MIT
+    ```
+
+3. (Optional) If optimization is enabled during compilation, take note of the value of the optimization runs parameter. 
+
+4. (Optional) If the contract constructor method accepts arguments, take note of the [ABI-encoded form](https://docs.soliditylang.org/en/develop/abi-spec.html) of the constructor arguments. 
+
+5. After deployment, take note the deployed contract address of the smart contract. The deployment address of the contract can be found either in the console output if using a command line based tool such as Truffle, Hardhat, or an Ethereum library, or it can be copied from the GUI in tools such as Remix IDE. 
+
+![Example Compiler Options in Remix IDE](/images/builders/tools/explorers/verify-contract/verify-contract-1.png)
+
+## Verify the Contract  {: #verifying-the-contract }
+
+The next step will be verifying the smart contract in an EVM compatible explorer for the Moonbeam network that you deployed to. 
+
+### Verify on Moonscan {: #verify-on-moonscan }
+
+Take the following steps to verify the contract on Moonscan: 
+
+1. Go to the [Verify & Publish Contract Source Code](https://moonbase.moonscan.io/verifyContract) page of Moonscan. 
+
+2. Fill in the contract's deployed address in the first field including the `0x` prefix.
+
+3. Select the compiler type. For the current `Incrementer.sol` example, select `Solidity (Single File)`.
+
+4. After selecting the compiler type, select the compiler version used to compile the contract. If the compiler version used was a nightly commit, uncheck the box under the field to select the nightly version. 
+
+5. Select the Open Source license used. If there was none used, select `No License (None)`.
+
+6. Click the `Continue` button at the bottom of the form to continue on to the next page.
+
+    ![First Page Screenshot](/images/builders/tools/explorers/verify-contract/verify-contract-2.png)
+
+7. On the second page, the `Contract Address`, `Compiler`, and `Constructor Arguments` fields should be prefilled. Copy and paste the entirely of the contract's content into the text field labeled as such. 
+
+8. Select `Yes` for `Optimization` if it was enabled during compilation, and filled in the number of runs under `Misc Settings/Runs(Optimizer)`. 
+
+9. (Optional) Add contract libraries and their addresses if any were used in the contract. 
+
+10. Check any other optional fields that may apply to your contract, and fill them out accordingly. 
+
+11. Click on the CAPTCHA at the bottom and the `Verify and Publish` button to confirm and begin verification.
+
+    ![Second Page Screenshot](/images/builders/tools/explorers/verify-contract/verify-contract-3.png)
+
+12. After a short wait, the result of verification will be displayed in the browser, and a success result page will display the contract's ABI encoded constructor arguments, the contract name, bytecode, and ABI. 
```

Comment:
same comment as above

Reply:
also UI elements should be in bold.... so `Solidity (Single File)` should be **Solidity (Single File)** and so on